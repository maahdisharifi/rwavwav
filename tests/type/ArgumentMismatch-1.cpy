void print1(int a, int b):
    int c = a + b
    printf(c)
end

void print2(float a, float b):
    int c = a + b
    printf(c)
end

int main():
    int a = 123
    float b = 5.5
    print1(a, b)
    print2(a, b)
    return 0
end