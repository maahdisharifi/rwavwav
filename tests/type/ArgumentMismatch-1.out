Line:7 -> argument type mismatch
Line:14 -> argument type mismatch
